packages/frontend/src/components/NavBar/DashboardExplorerBanner.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button, Text, Tooltip } from '@mantine/core';
+import { IconInfoCircle } from '@tabler/icons-react';
+import { useEffect, useMemo, type FC } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import useDashboardStorage from '../../hooks/dashboard/useDashboardStorage';
+import MantineIcon from '../common/MantineIcon';
+
+type Props = {
+    projectUuid: string;
+};
+
+export const DashboardExplorerBanner: FC<Props> = ({ projectUuid }) => {
+    const history = useHistory();
+    const { savedQueryUuid, mode } = useParams<{
+        savedQueryUuid: string;
+        mode?: string;
+    }>();
+
+    const { getEditingDashboardInfo, clearDashboardStorage } =
+        useDashboardStorage();
+    const { name: dashboardName, dashboardUuid } = getEditingDashboardInfo();
+
+    useEffect(() => {
+        window.addEventListener('unload', clearDashboardStorage);
+        return () =>
+            window.removeEventListener('unload', clearDashboardStorage);
+    }, [clearDashboardStorage]);
```

Comment:
ok im little bit lost here

Reply:
Same 😆 
This was here before, so it clears the dashboardStorage when the banner unloads. This usage is not recommended, but will address on a separate PR

Reply:
oh cool 😆